deephaven2 unit test failure in nightlies

```
  ======================================================================
  FAIL [0.004s]: test_datetime_at_midnight (test_time.DateT
  imeUtilsTestCase)
  ----------------------------------------------------------------------
  Traceback (most recent call last):
    File "/python/tests/test_time.py", line 68, in test_datetime_at_midnight
      self.assertGreaterEqual(diff_nanos(mid_night_time_ny, mid_night_time_pt), 0)
  AssertionError: -75600000000000 not greater than or equal to 0
```

--- a/pyintegration/tests/test_time.py
+++ b/pyintegration/tests/test_time.py
@@ -62,10 +62,22 @@ class DateTimeUtilsTestCase(unittest.TestCase):
         self.assertGreaterEqual(diff_nanos(dt, dt1), 100000000)
 
     def test_datetime_at_midnight(self):
-        dt = now()
+        datetime_str = "2021-12-10T02:59:59"
+        timezone_str = "NY"
+        dt = to_datetime(f"{datetime_str} {timezone_str}")
         mid_night_time_ny = datetime_at_midnight(dt, TimeZone.NY)
         mid_night_time_pt = datetime_at_midnight(dt, TimeZone.PT)
-        self.assertGreaterEqual(diff_nanos(mid_night_time_ny, mid_night_time_pt), 0)
+        self.assertEqual(diff_nanos(mid_night_time_ny, mid_night_time_pt)//10**9, -21*60*60 )
+
+        # DST ended in NY but not in PT
+        datetime_str = "2021-11-08T02:59:59"
+        timezone_str = "NY"
+        dt = to_datetime(f"{datetime_str} {timezone_str}")
+        mid_night_time_ny = datetime_at_midnight(dt, TimeZone.NY)
+        mid_night_time_pt = datetime_at_midnight(dt, TimeZone.PT)
+        print(nanos(mid_night_time_ny))
+        print(nanos(mid_night_time_pt))
+        self.assertEqual(diff_nanos(mid_night_time_ny, mid_night_time_pt)//10**9, -22*60*60 )
 
     def test_day_of_month(self):
         dt = now()

